src/agent0/core/hyperdrive/policies/simple_lp.py

```diff
@@ -0,0 +1,172 @@
+            - If the PNL is above threshold, and is not improving over time, then the bot takes no action.
+            - If the PNL is below threshold, and is improving over time, then the bot takes no action.
+            - If the PNL is below threshold, and is not improving over time, then the bot pulls liquidity.
+        """
+        return super().describe(raw_description)
+
+    @dataclass(kw_only=True)
+    class Config(HyperdriveBasePolicy.Config):
+        """Custom config arguments for this policy."""
+
+        pnl_target: FixedPoint = FixedPoint("100")
+        """The target PNL for the bot, in base."""
+        delta_liquidity: FixedPoint = FixedPoint("100")
+        """How much liquidity to add or remove, depending on policy outcome, in base."""
+        minimum_liquidity_tokens: FixedPoint = FixedPoint("100")
+        """
+        Minimum liquidity the bot will provide.
+        It will keep this much liquidity in the pool, even if it is losing money.
+        """
+        lookback_length: int = 10
+        """
+        How many steps back to look when computing PNL progress over time.
+        Each time `action` is called is a step.
+        """
+
+    def __init__(
+        self,
+        policy_config: Config,
+    ):
+        """Initialize the bot.
+
+        Arguments
+        ---------
+        policy_config: Config
+            The custom arguments for this policy
+        """
+        super().__init__(policy_config)
+        assert policy_config.pnl_target > FixedPoint(0), "PNL target must be greater than zero."
+        self.policy_config = policy_config
+        self.pnl_history: list[tuple[FixedPoint, FixedPoint]] = []
+        self.total_base_spent: FixedPoint = FixedPoint(0)
+
+    def action(
+        self, interface: HyperdriveReadInterface, wallet: HyperdriveWallet
+    ) -> tuple[list[Trade[HyperdriveMarketAction]], bool]:
+        """Specify actions.
+
+        Arguments
+        ---------
+        interface: HyperdriveReadInterface
+            Interface for the market on which this agent will be executing trades (MarketActions).
+        wallet: HyperdriveWallet
+            The agent's wallet.
+
+        Returns
+        -------
+        tuple[list[MarketAction], bool]
+            A tuple where the first element is a list of actions,
+            and the second element defines if the agent is done trading.
+        """
+        # Get the current state of the pool & the bot's position
+        current_block = interface.get_current_block()
+        pool_state = interface.get_hyperdrive_state(current_block)
+        current_holding = wallet.lp_tokens * pool_state.pool_info.lp_share_price
+
+        # Need to be in the game to play it
+        if self.policy_config.minimum_liquidity_tokens > current_holding:
+            trade_amount = self.policy_config.minimum_liquidity_tokens - current_holding
+            return [add_liquidity_trade(trade_amount)], False
```

Comment:
I think trade_amount is in base, so sounds like you want everything to be in base (not tokens? maybe you were thinking of base with _tokens, but I'd clarify, since I thought of lp_tokens)